### ISSUE TITLE
More of a question, Gacha Crystals and way to change the loot 

### ISSUE THREAD

**Issue by cyberangel67** [thommcgrath/Beacon#197]
As the title says, is there any way to change the loot from a gacha crystal?

**Comment by thommcgrath** [comment]
I think I recall a user having some luck with that, but I honestly don't know how.

**Comment by cyberangel67** [comment]
Yeah been looking for the codes to do it, but not having the Devkit or if it is even in there, makes finding such things hard.
